components/ProjectForm/index.js

```diff
@@ -88,7 +85,7 @@ export default function ProjectForm() {
       <StyledForm onSubmit={handleSubmit(onSubmit)}>
         <h2>Create new project</h2>
         <StyledLabel>
-          Project title
+          <p>Project title</p>
```

Comment:
Same as mentioned above, regarding the `paragraph element` as a child of a `label element`.
This applies to all other labels. 🙂